pyproject.toml

```diff
@@ -27,7 +27,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ensembl-genomio"
 version = "0.1" # should be assigned dynamically in the future
-requires-python = ">= 3.7"
+requires-python = ">= 3.7" # don't forget to update /cicd/gitlab/dot.gitlab-ci.yml
```

Comment:
Minimum version should say 3.8. I guess we forgot to apply this to this branch :see_no_evil:

Reply:
we forgot. let's wait until merge with an up-to-date